src/core/api/services_api/v1/_base_policy.xml

```diff
@@ -0,0 +1,19 @@
+<policies>
+    <inbound>
+        <base />
+        <choose>
+          <when condition="@(context.User.Groups.All(g =&gt; g.Name != &quot;ApiNoRateLimit&quot;))">
+            <rate-limit calls="150" renewal-period="5" remaining-calls-header-name="x-rate-limit-remaining" retry-after-header-name="x-rate-limit-retry-after" />
+          </when>
+        </choose>
```

Comment:
```suggestion
        <choose>
            <when condition="@(context.User.Groups.Select(g => g.Id).Contains("ApiNoRateLimit"))" />
            <when condition="@(context.User.Groups.Select(g => g.Id).Contains("ApiRateLimit250"))">
                <rate-limit calls="250" renewal-period="5" remaining-calls-header-name="x-rate-limit-remaining" retry-after-header-name="x-rate-limit-retry-after" />
            </when>
            <otherwise>
                <rate-limit calls="150" renewal-period="5" remaining-calls-header-name="x-rate-limit-remaining" retry-after-header-name="x-rate-limit-retry-after" />
            </otherwise>
        </choose>
```
more clean and also is ready for premium fast customer

from
https://github.com/pagopa/pagopa-infra/blob/3f81eafc12ef11395a3c6a152d68d644ba8282e4/src/core/api/checkout/checkout_payment_activations_auth/v1/_base_policy.xml.tpl#L5

Reply:
I'd add this in a future refinement, I'd like to import policies as they are, for now.